pydle/features/ircv3/tags.py

```diff
@@ -61,20 +61,24 @@ def parse(cls, line, encoding=pydle.protocol.DEFAULT_ENCODING):
                 if TAG_VALUE_SEPARATOR in raw_tag:
                     tag, value = raw_tag.split(TAG_VALUE_SEPARATOR, 1)
                 else:
+                    # Valueless or "missing" tag value
                     tag = raw_tag
```

Comment:
Else block needs to assign to `value`, otherwise line 66 may reference the name before its assigned to

Reply:
My bad, that's what I get for working too quickly. 😅

Reply:
I opted to just initialize `value` to `None` at the start of the loop in order to keep the single empty/missing normalization assignment. Of course, I can apply your suggestion instead if you prefer.